composer.json

```diff
@@ -22,13 +22,13 @@
     }
   ],
   "require": {
-    "php": "^7.2|^8.0",
-    "illuminate/encryption": "^6.0|^7.0|^8.0"
+    "php": "^7.4|^8.0",
+    "illuminate/encryption": "^8.75|^9.0"
   },
   "require-dev": {
     "mockery/mockery": "^1.3.3",
-    "orchestra/testbench": "^4.0|^5.0|^6.0",
-    "phpunit/phpunit": "^7.5|^8.0"
+    "orchestra/testbench": "^6.0|^7.0",
+    "phpunit/phpunit": "^8.5.23|^9.0"
```

Comment:
```suggestion
    "phpunit/phpunit": "^8.5|^9.0"
```

Reply:
This 8.5.23 fixed https://github.com/hihaho/laravel-encryptable-trait/runs/7711346579?check_suite_focus=true, see
https://stackoverflow.com/a/70974793